doc-content/enterprise-only/springboot/bus-app-import_proc.adoc

```diff
@@ -0,0 +1,33 @@
+[id='bus-apps-import_{context}']
+= Importing business assets projects into and deploying from Business Central and Decision Central
+
+You can import a business assets project that is part of a Red Hat Business Automation business application into Business Central or Decision Central and then deploy that project to a running service instance.
+
+.Prerequistes
+* You have a `<business-application>.zip` file that contains a business assets project, that you created using the http://start.jbpm.org[business applications] web site.
+* {PRODUCT} {CENTRAL} is installed.
```

Comment:
I think that another prerequisite is that Business Central is running and our business application is started in a developer mode - without this mode we cannot connect business application to Business Central.